site/README.md

```diff
@@ -0,0 +1,54 @@
+# aip.dev static site generator
+
+Welcome to our NIH static site generator.
+
+**Note:** If you do not care about this and just want to write or modify AIPs,
+see `CONTRIBUTING.md`.
+
+## Why?
+
+GitHub Pages normally automatically builds documentation with [Jekyll][], but
+as the AIP system has grown, we are beginning to reach the limits of what
+Jekyll can handle:
+
+- AIP adoption is handled through fork-and-merge and top-down configuration
+  files will lead to repetitive merge conflicts.
```

Comment:
I'm going to play Devil's advocate on these:

Does this not suggest that the core of the problem is in that infrastructure, the AIPs, and the configuration are too tightly coupled together?

Could we better separate these three aspects in the repository (either by directory structure or even splitting repos) in such a way that it would reduce merge conflicts?

Reply:
I think we should (in a followup) separate infrastructure from AIPs/configuration.
I think we would still have the fundamental issue though.